boards/arm/msp_exp432p401r_launchxl/doc/index.rst

```diff
@@ -26,7 +26,7 @@ Features:
 * Backchannel UART through USB to PC
 
 Details on the MSP-EXP432P401R LaunchXL development board can be found in the
-`MSP-EXP432P401R LaunchXL User's Guide`_.
+MSP-EXP432P401R LaunchXL User's Guide.
```

Comment:
we should probably tag this board as deprecated/remove it

Reply:
sure, but that is beyond the scope of this PR.